vnext/Desktop/Sandbox/SandboxJSExecutor.h

```diff
@@ -72,9 +72,7 @@ class SandboxJSExecutor : public JSExecutor {
 
   virtual void loadApplicationScript(
       std::unique_ptr<const JSBigString> script,
-      uint64_t scriptVersion,
-      std::string sourceURL,
-      std::string &&bytecodeFileName) override;
+      std::string sourceURL) override;
```

Comment:
>std::string sourceURL [](start = 6, length = 21)

why are we not passing by & here?

Reply:
This API is controlled by Facebook and unfortunately we can't change it without involving them. I think the correct way to express the semantics that Facebook intends is to use std::string&&, but from what I could tell Facebook's code avoids using rvalue reference parameters.

---
In reply to: [342744644](https://github.com/microsoft/react-native-windows/pull/3591#discussion_r342744644) [](ancestors = 342744644)

Reply:
Yep, left some discussion about this in the RN change.

---
In reply to: [342860718](https://github.com/microsoft/react-native-windows/pull/3591#discussion_r342860718) [](ancestors = 342860718,342744644)